File: staffOC/staffOC/model/Zjson.m
//
//  Zjson.m
//  staffOC
//
//  Created by zm004 on 16/7/28.
//  Copyright © 2016年 zm004. All rights reserved.
//

#import "Zjson.h"
@implementation Zjson
-(instancetype)initWith:(NSDictionary *)json{
    self.items = json[@"items"] != nil ? (NSArray *)json[@"items"] : [[NSArray alloc]init];
    self.success = json[@"success"] != nil ? (Boolean)json[@"success"] : false;
    self.addon = json[@"addon"] != nil ? (NSDictionary *)json[@"addon"] : [[NSDictionary alloc]init];
    self.detail = json[@"detail"] != nil ? (NSString *)json[@"detail"] : [[NSString alloc]init];
    self.code = json[@"code"] != nil ? (NSInteger)json[@"code"] : 0;
    self.totalCount = json[@"totalCount"] != nil ? (NSInteger)json[@"totalCount"] : 0;
    return self;
}
@end

File: netDemo/netDemo/utils/Zjson.swift
//
//  Zjson.swift
//  eshangk-ios
//
//  Created by zm002 on 14/12/30.
//  Copyright (c) 2014年 zm002. All rights reserved.
//

import UIKit

class Zjson {
   
    var items = [JSON]()
    var success = false
    var addon = JSON("{}")
    var detail = ""
    var code = 200
    var totalCount = 0
    
    var json:JSON

    init(json:JSON) {
        self.json = json
        items = json["items"].arrayValue
        success = json["success"].boolValue
        addon = json["addon"]
        detail = json["detail"].stringValue
        code = json["code"].intValue
        totalCount = json["totalCount"].intValue
    }
}
